src/app/forms/templates/test-records/section-templates/test/test-section-group1.template.ts

```diff
@@ -120,7 +120,7 @@ export const TestSectionGroup1: FormNode = {
               disabled: true,
               label: 'Anniversary date',
               viewType: FormNodeViewTypes.DATE,
-              editType: FormNodeEditTypes.DATE
+              editType: FormNodeEditTypes.HIDDEN
```

Comment:
Do we not want to still display these even though in amend mode? I understand not showing the field for contigency, not sure about hiding it in amend, especially if it's visible in view mode

Reply:
My point is that it's a field that shouldn't be editable because it's allegedly updated on the back end and that it should be visible in view mode as it's data related to the test record but the user shouldn't be able to manually change it.

Reply:
Heres my thoughts:

1. if its not editable but the value here would benefit editing others, it should still be viewable on the screen

2. The other thing is, with this missing would it confuse the user / make them think the record has missing data ? and result in a false negative bug being reported ?

Just my thoughts

Reply:
I guess those are reasons we could have it displayed but still marked as disabled, just wary of us having disabled fields viewable on the edit form so far. I'll change it back for the amend fields for now then.